site/ja/tutorials/load_data/tf-records.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<blockquote>各値は上記の関数を使って3種類の互換性のある型をからなる</blockquote><p>Typo?: 型をからなる</p><p>Maybe: 型のうちの1つだけを含む</p><p><br></p><blockquote>#2で作成したマップを<a href="https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/feature.proto#L85" target="_blank">特徴量メッセージ</a>に変換します。</blockquote><p>"Features メッセージ" seems better than "特徴量メッセージ" because</p><p>Features message is defined in "<a href="https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/feature.proto#L85" target="_blank">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/feature.proto#L85</a>".</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>